### PR TITLE
CLDR-14251 Modernize: more front end without dojo

### PR DESCRIPTION
**Pull request by btangmu**
-Reports, Forum, About, AdminPanel are now mostly working without dojo

-New cldrAbout.js and AboutST.java make the About page instead of about.jsp

-New cldrAdmin.js has Admin Panel code from old survey.js

-New AdminPanel.java replaces AdminAjax.jsp and AdminPanel.jsp

-New RecentActivity.java and cldrRecentActivity.js replace myvotes.jsp

-Move showForumStuff, etc., to new cldrForumPanel.js

-Move showInPop, etc., to new cldrInfo.js

-Move createChunk, etc., to new cldrDom.js

-Move showARIDialog, etc., to new cldrRetry.js

-Move oldvotes code to new cldrOldVotes.js

-Move loadMail code to new cldrMail.js

-Move report code to new cldrReportDash.js, cldrReportDates.js, cldrReportNumbers.js, cldrReportZones.js

-Move loadLocales code to new cldrLocales.js

-New unit tests TestCldrAbout.js, TestCldrAdmin.js, TestCldrCreateLogin.js

-Move cldrText.subVerbose into TestCldrText.js

-Remove Toggle Sidebar from the gear menu

-Show About in the gear menu even if not logged in

-Avoid nested functions; reduce function length by using subroutines

-Take advantage of ES6 Object Property Value Shorthand for IIFE return

-Remove dead code; clean up

-Comments

##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-_____
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: tools/cldr-apps/src/main/webapp/js/new/cldrLoad.js [hunk 1]
```diff
@@ -58,80 +53,78 @@ const cldrLoad = (function () {
...
 
     pucontent.appendChild(
-      cldrSurvey.createChunk(cldrText.get("itemInfoBlank"), "i")
+      cldrDom.createChunk(cldrText.get("itemInfoBlank"), "i")
     );
 
     // click on the title to copy (permalink)
-    cldrSurvey.clickToSelect(document.getElementById("ariScroller"));
-    cldrSurvey.updateIf(
+    const scroller = document.getElementById("ariScroller");
```
Comment (btangmu) [inline tools/cldr-apps/src/main/webapp/js/new/cldrLoad.js:100]: dead code -- ariScroller no longer exists (without dojo)

File: tools/cldr-apps/src/main/webapp/js/new/cldrLoad.js [hunk 2]
```diff
@@ -925,1366 +813,852 @@ const cldrLoad = (function () {
...
+  function getSpecial(str) {
+    const map = {
+      about: cldrAbout,
+      createAndLogin: cldrCreateLogin,
+      forum: cldrForum,
+      locales: cldrLocales,
+      mail: cldrMail,
+      oldvotes: cldrOldVotes,
+      r_compact: cldrReportNumbers,
+      r_datetime: cldrReportDate,
```
Comment (btangmu) [inline tools/cldr-apps/src/main/webapp/js/new/cldrLoad.js:873]: mismatch between the object and file names: "cldrReportDate" vs. "cldrReportDates" -- harmless but I'll add the "s" to the object name for consistency

File: tools/cldr-apps/src/main/webapp/myvotes.jsp [hunk 3]
```diff
@@ -52,9 +52,7 @@ showRecent('submitItems',null,'<%= user %>');
 ...
 
 <script>
-<% if (SurveyTool.USE_DOJO) %>
 showAllItems('allMyItems','<%= user %>');
-<% else %>
 cldrSurvey.showAllItems('allMyItems','<%= user %>');
```
Comment (btangmu) [inline tools/cldr-apps/src/main/webapp/myvotes.jsp:56]: Oops, I should have deleted the "cldrSurvey.showAllItems..." line; myvotes.jsp is unused if !USE_DOJO
